README.md

```diff
@@ -1,7 +1,11 @@
 # SysId: System Identification for Robot Mechanisms
+[![Build](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml/badge.svg)](https://github.com/wpilibsuite/sysid/actions/workflows/build.yml)
 
 This is the C++ version of [frc-characterization](https://github.com/wpilibsuite/frc-characterization). It uses the [wpimath](https://github.com/wpilibsuite/allwpilib/tree/main/wpimath) backend for generating feedforward and feedback gains.
 
+## Downloading and Running Development Versions of SysID
```

Comment:
Replace occurrences of `SysID` with `SysId` -- this is the proper capitalization.

Reply:
Done!